### ISSUE TITLE
Credentials section not expanding tilde for home directory

### ISSUE THREAD

**Issue by cwilkes** [serverless/serverless-google-cloudfunctions#51]
In the "provider" section there's "credentials" and the example lists "~/.gcloud/keyfile.json"  I tried and got a "ENOENT: no such file or directory, open" error.

Changing that to the full absolute path worked.   I realize this is probably a bug with the serverless framework itself, not your code but it shouldn't list an example that doesn't work out of the box.

  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       7.7.4
     Serverless Version: 1.9.0

**Comment by pmuens** [comment]
Thanks for reporting @cwilkes 👍 

Yes, that needs some more documentation.

**Comment by jtslear** [comment]
I don't think this is something that the overlying serverless framework is responsible as credentials for gcloud are stored differently than other providers.

https://github.com/serverless/serverless-google-cloudfunctions/blob/master/provider/googleProvider.js#L75

Unless this line gets updated to support tilde expansion, one could follow the trick of env variables, example:

`home.js`
```js
module.exports.theGoogleCloudCrednetialsFile = () => {
  return `process.env['HOME']/.gcloud/keyfile.json`
}
```

`serverless.yml`
```yaml
provider:
   credentials: ${file(./home.js):theGoogleCloudCredentialFile}
```

**Comment by pmuens** [comment]
Thanks again for reporting! And thanks @jtslear for the nice idea to fix this!

The issues was addressed and is fixed with #69
